jupyterlab/labbox_ephys_widgets_jp/src/extensions/curation/CurationSortingView/ControlPanel.tsx

```diff
@@ -0,0 +1,112 @@
+import { Grid, Paper } from '@material-ui/core';
+import React, { FunctionComponent, useCallback } from 'react';
+import { SortingCuration, SortingCurationDispatch } from '../../extensionInterface';
+
+type Props = {
+    width: number
+    curation: SortingCuration
+    curationDispatch: SortingCurationDispatch
+    selectedUnitIds: number[]
+}
+
+const buttonStyle = {
+    paddingTop: 3, paddingBottom: 3, border: 1, borderStyle: 'solid', borderColor: 'gray'
+}
+
+const ControlPanel: FunctionComponent<Props> = ({ width, selectedUnitIds, curation, curationDispatch }) => {
```

Comment:
curation and curationDispatch is the state/dispatch pair that controls curation for this particular sorting. Note that we send limited props to this control panel - it doesn't get the entire sorting. This is to increment modularity.